src/main/scala/org/globalforestwatch/summarystats/gfwpro_dashboard/GfwProDashboardGridSources.scala

```diff
@@ -14,30 +13,19 @@ case class GfwProDashboardGridSources(gridTile: GridTile, kwargs: Map[String, An
   val treeCoverDensity2000 = TreeCoverDensityPercent2000(gridTile, kwargs)
   val sbtnNaturalForest: SBTNNaturalForests = SBTNNaturalForests(gridTile, kwargs)
   val jrcForestCover: JRCForestCover = JRCForestCover(gridTile, kwargs)
+  val gadmAdm0: GadmAdm0 = GadmAdm0(gridTile, kwargs)
+  val gadmAdm1: GadmAdm1 = GadmAdm1(gridTile, kwargs)
+  val gadmAdm2: GadmAdm2 = GadmAdm2(gridTile, kwargs)
 
   def readWindow(
                   windowKey: SpatialKey,
                   windowLayout: LayoutDefinition
                 ): Either[Throwable, Raster[GfwProDashboardTile]] = {
 
-    for {
-      // Integrated alerts are Optional Tiles, but we keep it this way to avoid signature changes
-      integratedAlertsTile <- Either
-        .catchNonFatal(integratedAlerts.fetchWindow(windowKey, windowLayout))
-        .right
-      tcd2000Tile <- Either
-        .catchNonFatal(treeCoverDensity2000.fetchWindow(windowKey, windowLayout))
-        .right
-      sbtnNaturalForestTile <- Either
-        .catchNonFatal(sbtnNaturalForest.fetchWindow(windowKey, windowLayout))
-        .right
-      jrcForestCoverTile <- Either
-        .catchNonFatal(jrcForestCover.fetchWindow(windowKey, windowLayout))
-        .right
-    } yield {
-      val tile = GfwProDashboardTile(integratedAlertsTile, tcd2000Tile, sbtnNaturalForestTile, jrcForestCoverTile)
-      Raster(tile, windowKey.extent(windowLayout))
-    }
+    val tile = GfwProDashboardTile(
```

Comment:
why this change?

Reply:
This is part of making the GFWProDashboard sources be lazy.  With the lazy changes (as I already did a while ago for FCD), a tile is not fetched until it is first actually needed at run-time by the GridVisitor for a specific location.  I needed to make GFWProDashboard sources be lazy like FCD, because we don't want to load any GADM raster tiles during GridVisitor unless we are specifically using raster GADM because the # of locations is small.  I.e. If we are using the vector GADM and doing the full intersect, we don't want to drag down its performance by also loading raster gadm tiles that we won't actually use.